woodwork/type_sys/utils.py

```diff
@@ -74,7 +74,7 @@ def list_logical_types():
         [{'name': ltype.__name__,
           'type_string': ltype.type_string,
           'description': ltype.__doc__,
-          'physical_type': ltype.pandas_dtype,
+          'physical_type': ltype.pandas_dtype,  # --> should we offer both pd dtype and backup?
```

Comment:
Probably wouldn't be for this PR, and definitely low priority, but might be useful to add a 'backup_physical_type' section. (I can't remember if we've had this conversation, so maybe there's a good reason for not adding it in)